src/pages/en/guides/content.md

```diff
@@ -11,15 +11,15 @@ Astro helps you author and present your content. You can write a blog post direc
 ## Writing Content
 
 In Astro, you can author your content in a variety of ways: 
-- In Markdown (`.md`) files, designed to make it easy to write rich text content.
+- In Markdown (`.md` or `.markdown`) files, designed to make it easy to write rich text content.
```

Comment:
@sarah11918 The list of supported markdown files got bigger. We now support	'.markdown',	'.mdown',	'.mkdn',	'.mkd',	'.mdwn' and	'.md'.
	Should I go like this to avoid listing all the supported extensions?
```suggestion
- In Markdown (`.md` or [alternative extensions](https://github.com/github/markup/blob/b865add2e053f8cea3d7f4d9dcba001bdfd78994/lib/github/markups.rb#L1)) files, designed to make it easy to write rich text content.
```

Reply:
I could also list all the extensions once, refer to them later as "alternative extensions" in other occurrences and link to the list

Reply:
I have no idea if this is a "good" site to link to, but this is more like what I'm thinking re: showing file extensions: https://file.tips/software/markdown  What do you think? @MoustaphaDev

Reply:
> 
Oh yeah that page is very REGEXY 😅, I'm looking for a nicer page.

Reply:
I'm cool with adding this extra text "or alternative extension" in this one place, with a link ONLY if we find a link we like, and then later, we can actually write a small section in `markdown-content` explaining which file formats we support, and then we just link to THAT list.

Or, I can write that right now, if you need.  😅

Reply:
OK, so let's go with:

> In Markdown (.md, .markdown or alternative extension) files, designed...

in this ONE spot only, and leave it intentionally vague for now. 

Then, we'll do a PR for the `markdown-content` page (you are welcome to, or I will!) that will
- include our own written section of the files we support
- update references as appropriate throughout, so it's true but not clunky
- (additionally, scan other pages that list out both `.md` and `.markdown` and decide what our standard go-to line will be for those.)
Sound good?

Reply:
> I have no idea if this is a "good" site to link to, but this is more like what I'm thinking re: showing file extensions: https://file.tips/software/markdown What do you think? @MoustaphaDev

I'm worried that someone reading the docs might use one of those less commonly used extensions like `.mdtxt` thinking that it's supported while it's not, we only support markdown file extensions supported by github (the first link I sent). I think we could go with your first suggestion but maybe list somewhere all the extensions supported.

Reply:
> I'm cool with adding this extra text "or alternative extension" in this one place, with a link ONLY if we find a link we like, and then later, we can actually write a small section in `markdown-content` explaining which file formats we support, and then we just link to THAT list.
> 
> Or, I can write that right now, if you need. sweat_smile

I like this idea

Reply:
> OK, so let's go with:
> 
> > In Markdown (.md, .markdown or alternative extension) files, designed...
> 
> in this ONE spot only, and leave it intentionally vague for now.
> 
> Then, we'll do a PR for the `markdown-content` page (you are welcome to, or I will!) that will
> 
> * include our own written section of the files we support
> * update references as appropriate throughout, so it's true but not clunky
> * (additionally, scan other pages that list out both `.md` and `.markdown` and decide what our standard go-to line will be for those.)
>   Sound good?

Let's go with this !